test/sagemaker_tests/mxnet/inference/conftest.py

```diff
@@ -119,6 +120,9 @@ def instance_type(request, processor):
 def accelerator_type(request):
     return request.config.getoption('--accelerator-type')
 
+@pytest.fixture(scope='session', name='multi_region_support')
```

Comment:
Please return `None` for the fixture `n_virginia_sagemaker_session` and `n_virginia_ecr_image` if `not multi_region_support`, as an input guard. This will ensure that regions in other partitions do not fail, looking for access to `us-east-1`.